package.json

```diff
@@ -60,6 +60,7 @@
     "deepmerge": "^2.1.1",
     "ethereum-ens": "^0.7.4",
     "idempotent-babel-polyfill": "^7.0.0",
+    "leap-core": "^0.28.3",
```

Comment:
Should this be pinned too?

Reply:
could we avoid adding this dependency, and funnel the leap-extended web3 instance through `fallbackProvider` into daparatus?
then we would have a higher chance getting the PR accepted in upstream? 😁

Reply:
`this.props.fallbackWeb3Provider` is given as a prop when Metamask is not installed.
When Metamask is installed, it's injected web3Provider is used. If Metamask was compatible to Leap, then we could just funnel `leap-core` through `fallbackWeb3Provider`. Otherwise, Metamask would just inject the correct and extended web3Provider.